[accel brake map]update readme (#755)
tier4/AutowareArchitectureProposal.iv: vehicle/accel_brake_map_calibrator/accel_brake_map_calibrator/README.md

The value of each cell in the map is gray at first, and it changes from blue to red as the number of valid data in the cell accumulates. It is preferable to continue the calibration until each cell of the map becomes close to red. In particular, the performance near the stop depends strongly on the velocity of 0 ~ 6m/s range and the pedal value of +0.2 ~ -0.4, range so it is desirable to focus on those areas.

### Diagnostics

> `accel brake map_calibrator` publishes diagnostics depends on calibration status.

Reviewer: ???
```suggestion
`accel brake map_calibrator` publishes diagnostics depending on calibration status.
```

> `accel brake map_calibrator` publishes diagnostics depending on calibration status.

Reviewer: The `accel brake map_calibrator` publishes diagnostics message depending on the calibration status.

> `accel brake map_calibrator` publishes diagnostics depending on calibration status.

Reviewer: ```suggestion
The `accel brake map_calibrator` publishes diagnostics message depending on the calibration status.
```

> When the diagnostics type is `WARN`, the accuracy of current accel/brake map may be low and the accuracy improvement is strongly expected by updating the maps.

Reviewer: ```suggestion
Diagnostic type `WARN` indicates that the current accel/brake map is estimated to be inaccurate. In this situation, it is strongly recommended to perform a re-calibration of the accel/brake map.
```

| Status                  | Diagnostics Type | Diagnostics message                        | Description                                         |
| ----------------------- | ---------------- | ------------------------------------------ | --------------------------------------------------- |
| No calibration required | `OK`             | "OK"                                       |                                                     |
| Calibration Required    | `WARN`           | "Accel/brake map Calibration is required." | The accuracy of current accel/brake map may be low. |

> This diagnostics status can be also checked on Ros topic.

Reviewer: ```suggestion
This diagnostics status can be also checked on the following ROS topic.
```

```sh
ros2 topic echo /accel_brake_map_calibrator/output/update_suggest
```

> When the diagnostics type is `WARN`, `True` is published on this topic.

Reviewer: ```suggestion
When the diagnostics type is `WARN`, `True` is published on this topic and the update of the accel/brake map is suggested.
```
